core/controllers/admin.py

```diff
@@ -631,3 +632,21 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
```

Comment:
I previously asked for the email text to include the username of the sender. This was marked as "Done" in a previous comment but it is not done.

Reply:
According to https://github.com/oppia/oppia/pull/8544#issuecomment-581568603 I thought that I am not supposed to bring the username to the frontend so updated only in the backend. I think I misunderstood but I have made the necessary changes.